macro.pfsi/src/Event-Move.cpp

```diff
@@ -69,6 +72,27 @@ e_move_returnHome::e_move_returnHome(const double tEvent_, human* h) :
     double trip_t = tEvent_ + R::rexp(1./h->get_trip_frequency());
     h->addEvent2Q(e_move_takeTrip(trip_t,dest_id,h));
 
+    /* vaccinate/treat upon travel */
+    if(h->get_tile()->get_patch(dest_id)->get_reservoir()){
```

Comment:
1. In terms of the simulation, does the reservoir sometimes not exist? 2. In terms of coding, an if--then without an else clause is, statistically speaking, often a sign of a fault in the code. One way to handle that is to always explain _why_ there isn't an else clause, if there isn't really an else clause. This comes from the Code Complete book.